### ISSUE TITLE
Choice of Training Setting Parameters

### ISSUE THREAD

**Issue by TalhaAhmed2000** [RGenDiff/open-litevae#3]
Hello,

Could you please state what your training configuration was like for e.g:

1) Maximum Epochs	
2) Stopping Epoch
3) Batch Size
4) Random Seed
5) Learning Rate
6) Training Set Size
7) Validation Set Size
8) Test Set Size
9) GPU Choice

Or any other important detail in preprocessing, training etc you deem important. 

Thank you for your time & consideration

**Comment by vasimr** [comment]
Thank you for your interest.

All relevant details are described in the readme other than the lr and batch size. These followed LiteVAE with a lr of 1e-4 and a batch size of 32. Training duration was done by steps, not epochs, as described in the ReadMe and in the LiteVAE paper. 

No validation set was used - this is not a common practice in generative images. 

Hope that helps!
